modules/serverless-create-kafka-namespaced-broker.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * serverless/eventing/brokers/kafka-broker.adoc
+
+:_content-type: CONCEPT
+[id="serverless-create-kafka-namespaced-broker-{context}"]
+= Creating a Knative broker for Apache Kafka that uses an isolated data plane
+
+:FeatureName: The Knative Broker implementation for Apache Kafka with isolated data plane
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+To create a `KafkaNamespaced` broker, you must set the `eventing.knative.dev/broker.class` annotation to `KafkaNamespaced`.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource are installed on your {product-title} cluster.
+
+* You have access to an Apache Kafka instance such as link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/amq_streams_on_openshift_overview/kafka-concepts_str#kafka-concepts-key_str[Red Hat AMQ Streams], and have created a Kafka topic.
+
+* You have created a project, or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
```

Comment:
```suggestion
* You have created or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
```

Reply:
I find it harder to parse when the two are combined, so just adding an extra comma to separate it out more

"You have created a project, or have access to a project, with the appropriate roles"